skt/executable.py

```diff
@@ -460,6 +461,50 @@ def cmd_report(cfg):
     reporter.report()
 
 
+def cmd_console_check(cfg):
+    """
+    Check the console logs for any traces.
+
+    Args:
+        cfg: A dictionary of skt configuration.
+    """
+    remove_oldresult(cfg.get('output_dir'), 'console_check.')
+    console_result_path = join_with_slash(cfg.get('output_dir'),
+                                          'console_check.result')
+    console_report_path = join_with_slash(cfg.get('output_dir'),
+                                          'console_check.report')
+
+    if not cfg.get('krelease') or not cfg.get('console'):
+        raise Exception('<krelease> or <console-url> parameter missing!')
+
+    console_logs = []
+    trace_list = []
+
+    for console_path_or_url in cfg.get('console'):
+        console_log = skt.console.ConsoleLog(cfg.get('krelease'),
+                                             console_path_or_url)
+
+        console_logs.append(console_log)
+        trace_list.append(console_log.gettraces())
+
+    if any(trace_list):
+        report_string = ''
+
+        for traces in trace_list:
```

Comment:
Shouldn't this be called `trace`, not `traces`? We already have `trace_list`, and its name suggest that each of its elements is a trace.

Reply:
`traces` is a list of traces and `trace_list` is a list of them. If you have better names for this kind of nesting please do share it (this was the best I could come up with) but I feel that calling it `trace` would be misleading in this case

Reply:
I see. A list of lists is a bit of an awkward situation for any human language, I expect, but it's still better to be clear. Please see if you could use something like `trace_list` and `trace_list_list`, or `traces` and `traces_list`, or something of that sort. Personally I would use the former.